docs/setup/settings.asciidoc

```diff
@@ -429,6 +429,12 @@ to display map tiles in tilemap visualizations. By default,
 override this parameter to use their own Tile Map Service. For example:
 `"https://tiles.elastic.co/v2/default/{z}/{x}/{y}.png?elastic_tile_service_tos=agree&my_app_name=kibana"`
 
+| `migrations.batchSize:`
+ | Defines the number of documents migrated at a time. The higher the value, the faster the Saved Objects migration process will be performed at the cost of higher memory consumption. Should the migration fail with a `circuit_breaking_exception`, you need to set a smaller `batchSize` value. *Default: `1000`*
+
+| `migrations.enableV2:`
+ | experimental[]. Enables the new Saved Objects migration algorithm. See <<upgrade-migrations>> for details. The setting is going to removed from `v8.0` when `migrations v2` becomes stable. Setting the value to `false` will cause Kibana to use the legacy migration algorithm, which shipped in 7.11 and earlier versions. *Default: `true`*
```

Comment:
V2 migrations isn't a breaking change and keeping both adds technical debt so I think we should remove it as soon as possible.

Right now we've only seen v2 migrations fail because of other bugs which caused unusual amounts of saved objects and even though we have the `enableV2: false` workaround we've been asking users to clean up the unnecessary objects instead.

So after 7.12.1 I don't think we would need to use v1 migrations as a backup anymore, but just incase we might decide to keep it in 7.13 and remove in 7.14 (worst case we can always postpone this to later).

If you agree, maybe we should also deprecate this configuration option? I can create an issue to track this for the future.